### DISCUSSION TITLE
Stupid request

### DISCUSSION THREAD

**Discussion by toxicroadkill** [Chia-Network/chia-blockchain#3172] [General]
Its very minor, but i think others would find this useful, when creating the plot, going through all 4 phases, when it prints the status, prefix the status message with the phase number, would make it a lot easier to see how well it's going...stupid request, but it would be a handy thing

**Comment by boner-cmd** [comment]
In a terminal, you can always just scroll up to see which phase you're in:

![image](https://user-images.githubusercontent.com/54084132/115265482-e474e180-a0fc-11eb-9ee8-ef3d895575a2.png)

Unless you mean printing the phase for every line? That would be a lot of reminders...

**Comment by toxicroadkill** [comment]
I am using a VNC connection so scrolling up, is not a option...decided to do VNC, that way, prevents stupidity of closing the terminal window and it stopping

perhaps a cli option to enable the messages?

the other issue with scrolling up, depending on the terminal, it may have already filled the buffer and not allow scrolling that far back

**Comment by boner-cmd** [comment]
For the buffer limit, this sounds like a good use for something like tee​. As for accidentally closing the terminal window, if you use screen​ then even if you close the window you won't lose the process. Either way, I think there are other shell solutions for this other than adding redundant information to literally hundreds of lines of output. I think I could even write you a shell script myself to tell you the current phase using tee, a temporary output file, and sed.
________________________________

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
